rope/base/prefs.py

```diff
@@ -140,6 +140,22 @@ class Prefs:
         """),
     )
 
+    import_aliases: List[Tuple[str, str]] = field(
```

Comment:
I'm adding an autoimport prefs table in #516 , can we move this there?

Reply:
@lieryan do you have a preference where the "import_aliases" option goes. I tried moving it, but I was having trouble with the nested Prefs. Specifically I wasn't able to set the prefs for testing [here](https://github.com/python-rope/rope/pull/730/files#diff-df9f0c01d988e152b43e28de4f0738927877cbc0ff39fc861dcc03a2a5235ea1L22-R29).